Makefile

```diff
@@ -55,11 +55,14 @@ cluster-prepare:
 	$(MAKE) -C third_party/cert-manager deploy
 	$(MAKE) -C third_party/registry deploy
 	$(MAKE) -C third_party/vault deploy
+	kubectl apply -f https://raw.githubusercontent.com/IBM/dataset-lifecycle-framework/master/release-tools/manifests/dlf.yaml
+
```

Comment:
how adding DLF impact the quick-start https://ibm.github.io/the-mesh-for-data/docs/setup/quickstart/ ?
if there are changes, can you add in a follow-on PR?

Reply:
No changes are needed. The quick start uses the read module only for reading directly from the data source.
Datasets are not allocated, copy module is not applied either.

Reply:
Opened https://github.com/IBM/the-mesh-for-data/issues/323 for logging compliance information.